integrations-and-supported-tools/pytorch/notebooks/Neptune_PyTorch_Support.ipynb

```diff
@@ -0,0 +1,490 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "accelerator": "GPU",
+    "colab": {
+      "name": "Neptune_PyTorch_Support.ipynb",
+      "provenance": [],
+      "collapsed_sections": [],
+      "toc_visible": true
+    },
+    "kernelspec": {
+      "display_name": "Python 3",
+      "name": "python3"
+    },
+    "language_info": {
+      "name": "python"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "THIzFFk81nme"
+      },
+      "source": [
+        "# Neptune + PyoTorch\n",
+        "\n",
+        "## Before you start\n",
+        "\n",
+        "### Install dependencies"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "54CFqKRANLK5"
+      },
+      "source": [
+        "!pip install --quiet neptune-client"
```

Comment:
can you add the fixed version of the libraries (torch/ neputne-client numpy etc) -> basically copy what you have in the requirements.txt